aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -159,10 +145,8 @@ Tensor& _sigmoid_out_cpu(Tensor& result, const Tensor& self) {
     return at::op##_out(self, self);                            \
   }                                                             \
   Tensor& _##op##_out_cpu(Tensor& result, const Tensor& self) { \
-    checkBackend(#op, {result}, Backend::CPU);                  \
-    assert_no_internal_overlap(result, #op);                    \
     result.resize_(self.sizes());                               \
-    return at::legacy::th::_th_##op##_out(result, self);        \
+    return at::legacy::th::_th_##op##_out(result, self);                    \
```

Comment:
Remove the extra spaces to align with the previous lines?

Reply:
Actually I didn't change this file, looks like this is from rebase? @ezyang